Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+/// <summary>
```

Comment:
Good description of the class. It's not overly complex but neither is the class it represents.